packages/ui-extensions/docs/surfaces/checkout/staticPages/using-polaris-web-components.doc.ts

```diff
@@ -114,8 +114,8 @@ const data: LandingTemplateSchema = {
             tabs: [
               {
                 title: 'JavaScript',
-                code: './examples/properties-vs-attributes-jsx-props.js',
```

Comment:
This was needed since we ignore js files when generating docs, thanks @andrewmcgov for flagging this.